fem/fe/fe_base.hpp

```diff
@@ -576,6 +576,13 @@ class FiniteElement
    virtual const DofToQuad &GetDofToQuad(const IntegrationRule &ir,
                                          DofToQuad::Mode mode) const;
 
+   /** @brief Return the mapping from lexicographic DOF ordering to DOFs on
+       local face @a face_id. */
+   /** Given the @a ith DOF (lexicographically ordered) on the face referenced
+       by @a face_id, face_map[i] gives the corresponding index of the DOF in
+       the element (also lexicographically ordered). */
+   virtual void GetFaceMap(const int face_id, Array<int> &face_map) const;
```

Comment:
The `brief` description is somewhat confusing -- maybe use something like this:
```c++
   /** @brief Return the mapping from lexicographic face DOFs to lexicographic
       element DOFs for the given local face @a face_id. */
```
It maybe a good idea to add a note that this is still "experimental" -- I'm not sure if the current form will remain the same for non-tensor-product elements.

You can also add a note to clarify that for L2 bases this is only useful (or correct) for "Gauss-Lobatto", "Bernstein", and other "closed" bases.

Reply:
Done in cdfb02271.